hhs_oauth_server/request_logging.py

```diff
@@ -68,30 +70,37 @@ def __str__(self):
         elif getattr(self.response, 'content', False):
             log_msg['size'] = len(self.response.content)
 
-        log_msg['user'] = str(get_user_from_request(self.request))
         log_msg['ip_addr'] = get_ip_from_request(self.request)
 
         access_token = getattr(self.request, 'auth', get_access_token_from_request(self.request))
 
         if AccessToken.objects.filter(token=access_token).exists():
             at = AccessToken.objects.get(token=access_token)
-            log_msg['app_name'] = at.application.name
-            log_msg['app_id'] = at.application.id
-            log_msg['dev_id'] = at.application.user.id
-            log_msg['dev_name'] = str(at.application.user)
             try:
-                log_msg['org_name'] = at.application.user.user_profile.organization_name
-            except Exception:
```

Comment:
NOTE:  Removed this line, since we don't need this field in the logging.


Found that this line was having an exception, so the access_token_hash was always missing from the logging.
It was producing:
```python
AttributeError: 'User' object has no attribute 'user_profile'
```